Cabal/Cabal.cabal

```diff
@@ -165,29 +165,18 @@ source-repository head
 flag bundled-binary-generic
   default: False
 
-flag old-directory
-  description:  Use directory < 1.2 and old-time
-  default:      False
-
 library
   build-depends:
-    array      >= 0.4     && < 0.6,
-    base       >= 4.6     && < 5,
-    bytestring >= 0.9.2.1 && < 0.11,
-    containers >= 0.4.2.1 && < 0.7,
-    deepseq    >= 1.3     && < 1.5,
-    filepath   >= 1.3     && < 1.5,
-    pretty     >= 1.1.1   && < 1.2,
-    process    >= 1.1.0.1 && < 1.7,
-    time       >= 1.4     && < 1.10
-
-  if flag(old-directory)
-    build-depends: directory >= 1.1.0.2  && < 1.2,
-                   process   >= 1.0.1.1  && < 1.1.0.2,
-                   old-time  >= 1.1      && < 1.2
-  else
-    build-depends: directory >= 1.2      && < 1.4,
-                   process   >= 1.1.0.2  && < 1.7
+    array      >= 0.4      && < 0.6,
+    base       >= 4.6      && < 5,
+    bytestring >= 0.10.0.0 && < 0.11,
+    containers >= 0.5.0.0  && < 0.7,
+    deepseq    >= 1.3      && < 1.5,
+    directory  >= 1.2      && < 1.4,
+    filepath   >= 1.3      && < 1.5,
```

Comment:
1.3.0.1... and so on.

Let's have stricttly what's with GHC-7.6: https://ghc.haskell.org/trac/ghc/wiki/Commentary/Libraries/VersionHistory

Reply:
I only touched the bounds where I was specifically removing a CPP hack. AFAIK the rest don't strictly need bumping, but I guess we might as well.